application/templates/deployment.yaml

```diff
@@ -141,6 +144,10 @@ spec:
         image: {{ $image }}
 
         imagePullPolicy: {{ .Values.deployment.image.pullPolicy }}
+        {{- with .Values.deployment.lifecycle }}
+        lifecycle:
+          {{- toYaml .Values.deployment.lifecycle | nindent 10 }}
```

Comment:
```suggestion
          {{- toYaml . | nindent 10 }}
```

Reply:
![image](https://github.com/user-attachments/assets/07e9898c-d715-45c9-8527-3fbd8f052082)

I think this is still to be applied. @rtest12